cloud/pkg/admissioncontroller/admit_ruleendpoint.go

```diff
@@ -0,0 +1,45 @@
+package admissioncontroller
+
+import (
+	"net/http"
+	"strings"
+
+	admissionv1beta1 "k8s.io/api/admission/v1beta1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/klog/v2"
+
+	rulesv1 "github.com/kubeedge/kubeedge/cloud/pkg/apis/rules/v1"
+)
+
+func admitRuleEndpoint(review admissionv1beta1.AdmissionReview) *admissionv1beta1.AdmissionResponse {
+	reviewResponse := admissionv1beta1.AdmissionResponse{}
+	var msg string
+	switch review.Request.Operation {
+	case admissionv1beta1.Create:
+		raw := review.Request.Object.Raw
+		ruleEndpoint := rulesv1.RuleEndpoint{}
+		deserializer := codecs.UniversalDeserializer()
+		if _, _, err := deserializer.Decode(raw, nil, &ruleEndpoint); err != nil {
+			klog.Errorf("validation failed with error: %v", err)
+			msg = err.Error()
+			break
+		}
+		reviewResponse.Allowed = true
+		klog.Info("admission validation passed!")
+	case admissionv1beta1.Delete, admissionv1beta1.Connect:
+		//no rule defined for above operations, greenlight for all of above.
+		reviewResponse.Allowed = true
+		klog.Info("admission validation passed!")
+	default:
+		klog.Infof("Unsupported webhook operation %v", review.Request.Operation)
+		msg = msg + "Unsupported webhook operation!"
```

Comment:
1. why not add `review.Request.Operation` to msg?
2. just `msg = "Unsupoorted.."`

Reply:
done